src/rt/sched/behaviour.h

```diff
@@ -11,6 +11,36 @@ namespace verona::rt
   using namespace snmalloc;
   class Cown;
 
+  struct Request
+  {
+    Cown* _cown;
+
+    static const size_t READ_FLAG = 1;
+
+    Request() : _cown(nullptr) {}
+    Request(Cown* cown) : _cown(cown) {}
+
+    Cown* cown()
+    {
+      return (Cown*)((size_t)_cown & ~READ_FLAG);
+    }
+
+    bool is_read()
+    {
+      return ((size_t)_cown & READ_FLAG);
```

Comment:
Not as critical as the other two, since the pointer is never regenerated, but potentially uintptr_t is still better to be used here.